leetcode/string.md

```diff
@@ -0,0 +1,24 @@
+#String
```

Comment:
Your pull request: LEETCODE-1022 Added reverse-vowels-of-a-string


Your commit: LEETCODE-1022 Added reverse-vowels-of-a-string

Your commit: LEETCODE-1022 Refactored reverse-vowels-of-a-string
Action of your comment must be from ['Added', 'Deleted']
Your commit: LEETCODE-1022 Refactored reverse-vowels-of-a-string
Action of your comment must be from ['Added', 'Deleted']